__tests__/test-utils/featureFlags.ts

```
@@ -0,0 +1,9 @@
+// Re-export from the real test-utils location to avoid duplication
+export { FALLBACK_FLAGS } from "@/test-utils/featureFlags";
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)